pkg/types/validation/installconfig.go

```diff
@@ -1345,3 +1354,17 @@ func validateReleaseArchitecture(controlPlanePool *types.MachinePool, computePoo
 
 	return allErrs
 }
+
+// supportedSingleNodePlatform indicates if the IPI Installer can be used to install SNO on
+// a platform.
+func supportedSingleNodePlatform(isSingleNodeOpenShift bool, p *types.Platform) bool {
```

Comment:
could possibly rename the isSingleNodeOpenShift parameter to bootstrapInPlace maybe.